public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderCorner.tsx

```diff
@@ -86,10 +86,10 @@ export class PanelHeaderCorner extends Component<Props> {
 
     return (
       <Tooltip content={content} placement="top-start" theme={theme} interactive>
-        <section className={className} onClick={onClick} aria-label={ariaLabel}>
+        <button className={className} onClick={onClick} aria-label={ariaLabel}>
```

Comment:
```suggestion
        <button type='button' className={className} onClick={onClick} aria-label={ariaLabel}>
```
I'd add explicit type here just in case.

Reply:
i had this locally and forgot to push 😅 thanks for spotting